docs/nidigital/class.rst

```diff
@@ -1581,12 +1581,12 @@ get_channel_names
                 -   A range using a hyphen—for example, "0-3"
                 -   A range using a colon—for example, "0:3 "
 
-                You can combine comma-separated lists and ranges that use a hyphen or colon. Both out-of-order and repeated indices are supported ("2,3,0," "1,2,2,3"). White space characters, including spaces, tabs, feeds, and carriage returns, are allowed between characters. Ranges can be incrementing or decrementing.
+                You can combine comma-separated lists and ranges that use a hyphen or colon. Both out-of-order and repeated indices are supported ("2,3,0", "1,2,2,3"). White space characters, including spaces, tabs, feeds, and carriage returns, are allowed between characters. Ranges can be incrementing or decrementing.
```

Comment:
weird

Reply:
Our Tech Writer insisted.

Reply:
And looking at it now, I agree with him.